### ISSUE TITLE
Quick Start: Rotating device can misalign notification of task completion

### ISSUE THREAD

**Issue by lmischner** [wordpress-mobile/WordPress-iOS#18493]
Expected behavior
Rotating the device when a notification is shown should not change the alignment of the notification.

Actual behavior
The text in the notification is no longer center aligned.

![IMG_8468](https://user-images.githubusercontent.com/103850310/166395492-d4c081af-7119-4780-bf81-26344747dae7.PNG)

Steps to reproduce the behavior
1. Rotate device so you're in landscape mode
2. From menu screen, select stats tour in Grow your audience section
3. Select (now indicated) Stats option
4. While notification is up, rotate device so it's in portrait mode
5. Note alignment of notification

Tested on iPhone XS, iOS 15.4.1, WPiOS 19.8

**Comment by hassaanelgarem** [comment]
Hey @lmischner 👋 
~~I am unable to reproduce this. I tried this on iPhone 8, iPhone 13 Pro, and iPhone 11 simulators (iOS 15.4). And also using a physical iPhone 13 Pro (iOS 15.4.1). I don't have an iPhone XS (physical or simulator). Could this be device-specific? Or are you able to reproduce this on other devices?~~

I was finally able to reproduce it. Not consistently though, but enough to understand the root cause. #18758 fixes this 👍 

One other thing I noticed from your screenshot is that the notice's background should be white, not gray 🤔 

<img src="https://user-images.githubusercontent.com/25306722/170606810-6b2f972d-df0b-49c8-8bb9-a6a9d2ec6109.png" width=200>
